tests/Rector/Contrib/Nette/Image/NetteEnvironmentRector/Wrong/wrong.php.inc

```diff
@@ -0,0 +1,3 @@
+<?php
```

Comment:
All looks good. Just rename the directory to `NetteImageRector`, so autoloading works and the namespace and PSR-4 is kept

Reply:
Thanks, I fixed it